tests/sourcekit-lsp-initialization-message

```diff
@@ -0,0 +1,11 @@
+Content-Length: 152
+
+{
+    "jsonrpc": "2.0",
+    "id": 0,
+    "method": "initialize",
+    "params": {
+        "capabilities": {},
+        "trace": "verbose",
+    }
+}
```

Comment:
Missing newline at the end of the file?

Reply:
I'll have to test if including one breaks the test. There's not supposed to be a newline at the end of the RPC message